src/routes/index.js

```diff
@@ -38,9 +39,10 @@ const apiVersion = process.env.API_VERSION;
 const baseUrl = `/api/${apiVersion}`;
 
 router.get('/', (req, res) => res.status(200).json({ status: 200, data: 'Welcome to Authors Haven.' }));
+router.use(article);
 router.use(baseUrl, api);
 router.use(oauth);
-router.use(article);
+router.use(likes);
```

Comment:
Why are we having likes route outside the baseurl format we are using for our API?
We are now having 10 routes instead of 5 you intended. 
ie those with (localhost:3000/api/v1/likes/.......) as per `line 20 in src/routes/api/index.route.js`
and those with (localhost:3000/likes/.......) as per `line 45 in src/routes/index.js`